_layouts/404.html

```diff
@@ -15,29 +15,28 @@
 {% block content_main %}
     <div class="error_copy_container">
         <header class="error_header u-mt30">
-            <h1 class="superheader">Oops! 404 error.</h1>
-            <p class="short_desc">Something’s unplugged. We can’t find the page you 
-            	were looking for, but here are a number 
-            	of other ways you might try to get there.
+            <h1 class="superheader">Something’s disconnected.</h1>
+            <p class="short_desc">We couldn’t find the exact page you 
+            were looking for, but we have other information that may be useful.
             </p>
         </header>
         <aside class="error_action_container padded-container u-mt60 u-mb60">
             <div class="content-l content-l__large-gutters padded-container_body">
                 <div class="content-l_col content-l_col-1-2">
                     <p>
-                        Explore our other useful tools and resources.
+                        Explore our tools and resources.
                     </p>
                     <a href="/" class="btn btn__full btn__error-home">Visit homepage</a>
                 </div>
                 <div class="content-l_col content-l_col-1-2 content-l_col__before-divider">
                     <p>
-                        If the problem persists please let us know.
+                        If you’re sure this is the right place, let us know.
                     </p>
                     <a class="jump-link jump-link__right" href="/contact-us">
-                        Visit contact us page
+                        Contact us
```

Comment:
Should we include the General Inquiries contact us link here rather than make people go to another page and look through all the offices only to most likely setting on General Inquiries?